src/core/src/sys/systems.rs

```diff
@@ -465,6 +464,19 @@ impl System {
         let mkind = self.particles[m].kind;
         self.interactions.dihedrals(ikind, jkind, kkind, mkind)
     }
+
+    /// Return the largest cutoff radius from pair interactions
+    /// excluding electrostatic interactions.
+    pub fn largest_cutoff(&self) -> f64 {
```

Comment:
I am not comfortable with having this as a `System` function. It feels like it belongs to `Interactions` instead, and should be accessed by `system.interactions().largest_cutoff()`

`System` already have too much functions =/

Aside, how often do you plan to use this ? If it is only for `Resize`, it could just be a piece of code inside resize, that we can refactor in a public function later if needed.